platform/raspberry_pi/rpi0/mods.conf

```diff
@@ -14,6 +14,7 @@ configuration conf {
 	@Runlevel(0) include embox.arch.arm.mmuinfo
 	
 	@Runlevel(0) include embox.arch.system(core_freq=100000000)
+	@Runlevel(0) include embox.driver.dma.bcm283x_dma
```

Comment:
I can not find bcm283x_dma. Maybe you have forgotten to commit some code?

Reply:
Oh shoot!  some pending changes got included.  It was just to be spaces.  I'm working towards supporting DMA here but that code isn't ready, the mailbox updates are a prerequisite.  I'll re-submit the whole thing as one commit with a forced push shortly.